### ISSUE TITLE
[data-import] Unable to insert custom metadata record using Metadata API Type

### ISSUE THREAD

**Issue by uditsinghal-salesforce** [tprouvot/Salesforce-Inspector-reloaded#367]
**Make sure to read the troubleshooting section before creating an issue**

- [x] I've read the [common issues](https://tprouvot.github.io/Salesforce-Inspector-reloaded/troubleshooting/) and the one I'm about to create is not one of those documented.

**Describe the bug**

**To Reproduce**
1. Go to Data Import
2. Change API type to Metadata
3. Action -> Upsert Metadata
4. Choose the desired custom metadata object
5. Paste the data to be inserted and update field mappings
6. Click on Run Upsert Metadata

**Expected behavior**
Metadata records should be imported correctly

**Screenshots**
<img width="1499" alt="Screenshot 2024-03-22 at 1 30 32 PM" src="https://github.com/tprouvot/Salesforce-Inspector-reloaded/assets/151709894/64ccdd3f-69e4-4473-a514-ccdd8c26fa54">
<img width="1504" alt="Screenshot 2024-03-22 at 1 30 41 PM" src="https://github.com/tprouvot/Salesforce-Inspector-reloaded/assets/151709894/aa2b98c8-a14f-4128-918a-a80dfa70832d">

**Desktop (please complete the following information):**
 - OS: MacOS
 - Browser: Chrome
 - Version: 123.0.6312.59

**Comment by tprouvot** [comment]
Hi @uditsinghal-salesforce 
Could you insert data manually, then perform the query with selected fields in data export and then compare it with the original data you want to insert ? 
The goal is top compare the data and check the differences

**Comment by tprouvot** [comment]
Hi @uditsinghal-salesforce,
Did you had a chance to see my last comment ?

**Comment by uditsinghal-salesforce** [comment]
Hi Thomas,

I have compared the data after inserting it and found one change. I was using the parent object name in DynamicScreenHeader__c during import which seems to be replaced by id after the metadata records were created manually. I tried replacing the name with the parent Id in import but still facing an error while importing custom metadata records. Attached a screenshot for your reference.

<img width="1506" alt="Screenshot 2024-04-03 at 9 56 30 AM" src="https://github.com/tprouvot/Salesforce-Inspector-reloaded/assets/151709894/392ecd96-a891-461b-8749-239c49bdf39d">
